.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
-  extends: ['plugin:prettier/recommended', 'next/core-web-vitals', 'prettier'],
```

Comment:
The `plugin:prettier/recommended` extension is just a shorthand that adds the `prettier` declarations that we're also adding manually